EPlast/EPlast/gulpfile.js

```diff
@@ -28,6 +29,14 @@ gulp.task('less', function () {
         .pipe(gulp.dest(paths.webroot + 'css/lessCompiled'));
 });
 
+gulp.task("sass", function () {
+    return gulp.src('wwwroot/sass/**/*.scss')
+        .pipe(sass())
+        .pipe(gulp.dest(paths.webroot + 'css/sassCompiled'));
+});
+
 gulp.task('build', gulp.parallel('scripts', 'less'));
 
+gulp.task('build', gulp.parallel('scripts', 'sass'));
```

Comment:
Why this is wrong
1) now you have 2 tasks with same name build
2) you would execute scripts task twice, if it doesn't give an error right away
What you need to do
create task styles which would run in parallel sass and less
update build task so it would run in parallel scripts task and styles task